### DISCUSSION TITLE
How long do Canary tokens last in the free version?

### DISCUSSION THREAD

**Discussion by OlesYudin** [thinkst/canarytokens#587] [Q&A] [answered]
I'm planning to add some Canary tokens as decoys in my infrastructure, but I don't expect them to be triggered often. Ideally, these decoys will never be activated, but mistakes can happen:)
My question is: Do Canary tokens in the free version have a TTL, and should I periodically regenerate them even if they haven't been triggered?

**Comment by thinkst-marco** [comment, marked as answer]
Canarytokens have no expiry so we don't advise regenerating them. We'll keep monitoring them for use, and send you the alert if that (unfortunately) ever happens.

**Reply by OlesYudin** [reply]
Thank you!
